### PR TITLE
add german

### PR DESCRIPTION
**Pull request by Marta108**

### CODE REVIEW

File: anrede.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hallo Welt!
```
Comment (betoDavila86) [inline anrede.txt]: Marta, add another exclamation mark ;)

Comment (Marta108) [inline anrede.txt]: add another exclamation mark

Comment (Marta108) [reply anrede.txt]: Resuelto Beto, gracias :)

File: anrede.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Hallo Welt!!
```
Comment (betoDavila86) [inline anrede.txt:1]: Ready to merge!

**Pull request merged**